server/.vscode/tasks.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Is this file created by default while setting up the boilerplate .NET API? If not, can you please remove the .vscode folder and add .vscode to the gitignore?

Reply:
Removed from git